content/release-10-15-0/devices-10-15-0-bundle/10-15-0-0.md

```diff
@@ -208,4 +209,21 @@ Considering the pool of connections to microservices has a limited size (default
 MTM-34911</td>
 </tr>
 
-</tbody></table></div>
+<tr>
```

Comment:
@atulpandasag The regular release notes entries are handled by the release notes fields in the Jira ticket and will be added by me from there. I will leave them here now, but in the future you only have to fill in the fields in the tickets.

Reply:
Sure. Thank you. I'll take care of it.